### PR TITLE
Brush up Lion's Shield

### PR DESCRIPTION
**Pull request by simonward**
Also choose better icon

Closes https://github.com/foundryvtt/pf2e/issues/15168

### CODE REVIEW

File: packs/equipment-effects/effect-lions-bite.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                "traits": [
+                    "deadly-d6"
+                ]
+            },
+            {
+                "key": "WeaponPotency",
+                "label": "PF2E.PotencyRuneLabel",
+                "selector": "{item|id}-attack",
+                "value": 1
+            }
```
Comment (TikaelSol) [inline packs/equipment-effects/effect-lions-bite.json:46]: It's not clear whether it should benefit from the potency rune here

Comment (simonward) [reply packs/equipment-effects/effect-lions-bite.json]: I think you're right about it not having the potency rune - fixed.

As for the WeaponPotency label, if it's not specified it defaults to the item name.
Screen Shot 2024-06-21 at 10 59 20 AM

**Comment by stwlam** [comment]
Surely there is a shield with a kitty cat somewhere :(

**Comment by simonward** [comment]
> Surely there is a shield with a kitty cat somewhere :(

I hoped so too, but couldn't find one.

**Pull request merged**